web/components/header.tsx

```diff
@@ -28,6 +32,37 @@ export const Header: React.FC<HeaderProps> = ({ signOut, setShowSidebar }) => {
   const showWallet = process.env.REACT_APP_SHOW_CONNECT_WALLET === 'true';
   const { doOpenAuth } = useConnect();
   const name = bnsName();
+  const contractAddress = process.env.REACT_APP_CONTRACT_ADDRESS || '';
+  const stxAddress = useSTXAddress();
+  const [isVotingOpen, setisVotingOpen] = useState(false);
+
+  useEffect(() => {
+    let mounted = true;
+
+    const getData = async () => {
+      const proposals = await callReadOnlyFunction({
+        contractAddress,
+        contractName: 'arkadiko-governance-v3-1',
+        functionName: 'get-proposals',
+        functionArgs: [],
+        senderAddress: stxAddress || '',
```

Comment:
REQ: this page will fail when the user is not logged in (e.g. try incognito)

The fix is to remove this `stxAddress` completely (also line 36 + the import) and just use `contractAddress` as the sender address

Reply:
Right, good catch. Will update.